pkg/sql/app_stats.go

```diff
@@ -113,6 +115,10 @@ func (a *appStats) recordStatement(
 	// Get the statistics object.
 	s := a.getStatsForStmt(stmtKey{stmt: buf.String(), failed: err != nil, distSQLUsed: distSQLUsed})
 
+	// Make sure to add statement recording to the overhead latency! The above
+	// FormatNode call is particularly expensive.
+	ovhLat += timeutil.Now().Sub(now).Seconds()
```

Comment:
1) in `executor_statement_metrics.go` move the `recordStatement` call above the block that increases things with `RecordValue()`
2) make `recordStatement` take the `phaseTimes` array as argument instead of the latencies
3) make `recordStatement` compute the latencies itself
4) add the extra overhead using the formula `timeutil.Now().Sub(phaseTimes[plannerEndExecStmt])` to avoid the extra call to `Now()` and capture a bit more overhead this way. Don't forget to add this extra latency to both "service" and "overhead". 
5) make `recordStatement` return the computed latencies, so that the remainder of the code in `recordStatementSummary` sees the updated latencies.